wp-content/plugins/ig-content-loader-base/plugin.php

```diff
@@ -0,0 +1,278 @@
+<?php
+/**
+ * Plugin Name: Content Loader Base
+ * Description: Template-base to include any foreign content into Integreat
+ * Version: 0.1
+ * Author: Julian Orth
+ * Author URI: https://github.com/Integreat
+ * License: MIT
+ */
+
+/**
+ * Register Cron Job Hook to update data every 12 hour 
+ */
+//register_activation_hook(__FILE__, 'cl_update_content');
+//
+//function cl_update_content() {
+//	if (! wp_next_scheduled ( 'cl_update_contents' )) {
+//	wp_schedule_event(time(), 'twicedaily', 'cl_update_contents');
+//	}
+//}
+//
+//add_action('cl_update_contents', 'cl_update_content');
```

Comment:
In this case I would like to leave it, because it can come in handy. We could just remove the hook, but that's not really making any difference.